src/style/style.js

```diff
@@ -1215,7 +1215,7 @@ class Style extends Evented {
             this.pauseablePlacement.continuePlacement(this._order, this._layers, layerTiles);
 
             if (this.pauseablePlacement.isDone()) {
-                this.placement = this.pauseablePlacement.commit(browser.now());
+                this.placement = this.pauseablePlacement.commit(browser.now(), transform.zoom);
```

Comment:
> If the zoom value is coming from the same transform that we passed to the PausablePlacement constructor in L1204, why do we need to add it as an argument for the commit method (here as well on Placement)? Same question for the stillRecent method as well. What am I missing?

Thanks for catching this. The two values were different (one is when placement starts, the other is when placement ends). But using the one from when it starts is more correct.

stillRecent needs the latest zoom so that it can calculate how much the zoom has changed.